pkg/ipns/record.go

```diff
@@ -0,0 +1,47 @@
+package ipns
+
+import (
+	"time"
+
+	"github.com/ipfs/go-ipns"
+	pb "github.com/ipfs/go-ipns/pb"
+	crypto "github.com/libp2p/go-libp2p-core/crypto"
```

Comment:
Is there a reason to be using libp2p crypto vs built in crypto?

Reply:
go-ipns requires keys from `go-libp2p-core`

Reply:
was able to convert keys over, think they wrapped this to add an equality operation to the interface -_-